frontend/src/components/nlp/components/NlpValueForm.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright © 2025 Hexastack. All rights reserved.
+ *
+ * Licensed under the GNU Affero General Public License v3.0 (AGPLv3) with the following additional terms:
+ * 1. The name "Hexabot" is a trademark of Hexastack. You may not use this name in derivative works without express written permission.
+ * 2. All derivative works must include clear attribution to the original creator and software, Hexastack and Hexabot, in a prominent location (e.g., in the software's "About" section, documentation, and README file).
+ */
+
+import { useRouter } from "next/router";
+import { FC, Fragment, useEffect } from "react";
+import { Controller, useForm } from "react-hook-form";
+
+import { ContentContainer, ContentItem } from "@/app-components/dialogs/";
+import { Input } from "@/app-components/inputs/Input";
+import MultipleInput from "@/app-components/inputs/MultipleInput";
+import { useCreate } from "@/hooks/crud/useCreate";
+import { useGet } from "@/hooks/crud/useGet";
+import { useUpdate } from "@/hooks/crud/useUpdate";
+import { useToast } from "@/hooks/useToast";
+import { useTranslate } from "@/hooks/useTranslate";
+import { EntityType, Format } from "@/services/types";
+import { ComponentFormProps } from "@/types/common/dialogs.types";
+import { INlpValue, INlpValueAttributes } from "@/types/nlp-value.types";
+
+export const NlpValueForm: FC<
+  ComponentFormProps<{ data: INlpValue; canHaveSynonyms: boolean }>
+> = ({ data: props, Wrapper = Fragment, WrapperProps, ...rest }) => {
+  const { data, canHaveSynonyms } = props || {};
+  const { t } = useTranslate();
+  const { toast } = useToast();
+  const { query } = useRouter();
+  const { refetch: refetchEntity } = useGet(data?.entity || String(query.id), {
+    entity: EntityType.NLP_ENTITY,
+    format: Format.FULL,
+  });
+  const options = {
+    onError: () => {
+      rest.onError?.();
+      toast.error(t("message.internal_server_error"));
+    },
+    onSuccess() {
+      rest.onSuccess?.();
+      toast.success(t("message.success_save"));
+    },
+  };
+  const { mutateAsync: createNlpValue } = useCreate(
```

Comment:
Are we able to use mutate instead ?
```suggestion
  const { mutate: createNlpValue } = useCreate(
```

Reply:
Thank you for the idea.
Yes it's possible to refactor the code and use the onSuccess callback function with a synchronous code